src/cfn_yaml/core.clj

```diff
@@ -98,14 +98,20 @@
       (throw (ex-info (str "Unresolved references found: " unresolved-references)
                       {:unresolved unresolved-references})))))
 
+(defn parse* [yml]
+  (yaml/decode (.load (make-yaml) yml) true))
+
 (defn parse [yml]
-  (let [cfn (yaml/decode (.load (make-yaml) yml) true)]
+  (let [cfn (parse* yml)]
     (validate-references cfn)
     cfn))
 
+(defn generate-string* [cfn-data]
```

Comment:
I'm ok with this, but what would you think about having another arity for specifying validation:

```clojure
(defn generate-string
  ([cfn-data]
   (generate-string cfn-data true))
  ([cfn-data validate-references?]
   (when validate-references?
     (validate-references cfn-data))
   (.dump (make-yaml) (yaml/encode cfn-data))))
```